src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -863,3 +877,41 @@ function getQuestionIdSuffix(questionNum) {
     return idSuffix;
 }
 
+function toggleCustomFeedbackPathsDisplay(toggleLink) {
```

Comment:
Can we put the feedback path related methods in `instructorEditPage/feedbackPath.js`?
The methods are separated into the said file in #5744, I think should do the same here so when merging these two in the future the feedback path related methods will not be split in two locations.